scripts/e

```diff
@@ -280,4 +282,25 @@ case "$1" in
     # Open the generated solution
     "$DEV_ENV" "$OUT_DIR/all.sln"
     ;;
+  update_local_lok)
+    OUT_DIR="$BASE_PATH/src/out/$ELECTRON_OUT_DIR"
+
+    if [ -z "$LOK_PATH" ]
+    then
+      echo "LOK_PATH not set. Defaulting to $DEFAULT_LOK_PATH"
+      LOK_PATH=$DEFAULT_LOK_PATH
+    else
+      echo "LOK_PATH was set to $LOK_PATH"
+    fi
```

Comment:
Can do this with one line:

`LOK_PATH="${LOK_PATH:-$DEFAULT_LOK_PATH}"`

In fact you can get rid of DEFAULT_LOK_PATH and just make it `LOK_PATH="${LOK_PATH:-../libreofficekit}"`